debug_toolbar/panels/sql/tracking.py

```diff
@@ -114,7 +117,7 @@ def _record(self, method, sql, params):
             _params = ''
             try:
                 _params = json.dumps([self._decode(p) for p in params])
-            except Exception:
+            except TypeError as e:
```

Comment:
I'm a bit surprised that this pull request passes the build, since this is an unused variable right here.

Reply:
Strange indeed. Just pushed a commit removing the variable.